client/client-multi/client-objective-c/Kaa/log/AbstractLogCollector.m

```diff
@@ -138,29 +144,31 @@ - (void)onLogResponse:(LogSyncResponse *)response {
         if (response.deliveryStatuses && response.deliveryStatuses.branch == KAA_UNION_ARRAY_LOG_DELIVERY_STATUS_OR_NULL_BRANCH_0) {
             BOOL isAlreadyScheduled = NO;
             NSArray *deliveryStatuses = response.deliveryStatuses.data;
+            
             __weak typeof(self) weakSelf = self;
+            
             for (LogDeliveryStatus *status in deliveryStatuses) {
-                
                 NSNumber *key = @(status.requestId);
-                
                 __block BucketInfo *bucketInfo = self.bucketInfoDictionary[key];
                 
                 if (bucketInfo) {
                     [self.bucketInfoDictionary removeObjectForKey:key];
 
                     if (status.result == SYNC_RESPONSE_RESULT_TYPE_SUCCESS) {
+	                    bucketInfo.receivedResponseTime = [NSDate currentTimeInMilliseconds];
+                        
                         [self.storage removeBucketWithId:status.requestId];
                         
                         [[self.executorContext getCallbackExecutor] addOperationWithBlock:^{
                             [weakSelf notifyOnSuccessDeliveryRunnersWithBucketInfo:bucketInfo];
+				            [self.deliveryRunnerDictionary removeObjectForKey:@(bucketInfo.bucketId)];
```

Comment:
As far as I see this method is just moved here out of notifyOnSuccessDeliveryRunnersWithBucketInfo: method.
Is there any reason for that?